Pages/Hotels/PropertyInfo.js

```diff
@@ -0,0 +1,137 @@
+class PropertyInfo {
+
+    // Webelements Locators
+
+    likeToListLocator = '//p[text()="What would you like to list?"]';
+    lodgingOptionLocator = '//p[text()="Lodging"]';
+    privateResidenceOptionLocator = '//div[@id="classification_privateResidence"]';
+    step1Of3Locator = '//div[@data-wdio="lyp-progress-bar"]';
+    step2Of3Locator = '//div[contains(text(), "Step 2")]';
+    howMuchEarnLocator = '//h1[contains(text()," could earn!")]';
+    increaseBedroomLocator = '//button[@aria-label = "Increase bedrooms"]';
+    increaseBathroomLocator = '//button[@aria-label = "Increase bathrooms"]';
+    propertyInfoNextButtonLocator = '//button[@id="propertyInfoNextBtn"]';
+    whereIsYourPropertyLocator = '//div[contains(text(), "Step 2")]/following-sibling::h1';
+    locationTypeLocator = '//input[@id="locationTypeAhead"]';
+    searchDropDownLocator = '//ul[@role="menu"]/child::li[1]';
+    graphLocator = '//section[@class="map-section location"]';
+    pinInGraphLocator = '//img[contains(@src, "maps.gstatic.com/")]';
+    moveThePinTextLocator = '//span[contains(text(), "adjust the location")]';
+
+
+    // Property Related Functions
+
+
+    async switchWindow() {
+
+        const allHandles = await browser.getWindowHandles();
+        for (const handle of allHandles) {
+            await browser.switchToWindow(handle);
+            const currentUrl = await browser.getUrl();
+            if (currentUrl.includes('expediapartnercentral')) {
+                break;
+            }
+        }
```

Comment:
You don't need this function if you parameterize the other `switchWindow()` function above.